README.md

```diff
@@ -32,14 +32,15 @@
 * ansible.cfg: User customized Ansible configuration file
 * autoinstall: Folder for guest OS unattend install configuration files
 * common: Folder for common tasks called in test cases
-* docs: Folder for guide files
+* docs: Folder for guide file and known issues
 * env_setup: Folder for playbooks or tasks which to prepare or cleanup testing environment
 * linux: Folder for playbooks to test Linux guest OS
 * Windows: Folder for playbooks to test Windows guest OS
 * plugin: Folder for plugin scripts
 * tools: Folder for 3rd-party tools used in test cases
 * vars: Folder for variable files used in testing
 * logs: Folder generated by default for files collected and logs generated in test run
```

Comment:
There is no logs folder from source code. Maybe we can add a step to tell user where to find logs in Steps to Launch Testing after running playbook.

Reply:
Removed this "logs" line, and added to steps 5 above. Thanks.